generated/nixnet/nixnet_service.cpp

```diff
@@ -166,6 +167,78 @@ namespace nixnet_grpc {
     }
   }
 
+  //---------------------------------------------------------------------
+  //---------------------------------------------------------------------
+  ::grpc::Status NiXnetService::ConvertFramesToSignalsSinglePoint(::grpc::ServerContext* context, const ConvertFramesToSignalsSinglePointRequest* request, ConvertFramesToSignalsSinglePointResponse* response)
+  {
+    if (context->IsCancelled()) {
+      return ::grpc::Status::CANCELLED;
+    }
+    try {
+      auto session_ref_grpc_session = request->session_ref();
+      nxSessionRef_t session_ref = session_repository_->access_session(session_ref_grpc_session.id(), session_ref_grpc_session.name());
+      auto frame_buffer = convert_from_grpc<u8>(request->frame_buffer());
```

Comment:
note: `frame_buffer` is of type `FrameBuffer` and uses same concept as `SockAddrInputConverter` for implicit conversion to `u8*` during library call and for the `size` calculation below.